src/index.ts

```diff
@@ -212,7 +212,7 @@ async function uploadProfile(profile: perftools.perftools.profiles.IProfile) {
 }
 
 // Could be false or a function to stop heap profiling
-let heapProfilingTimer: string | number | NodeJS.Timeout | undefined = undefined
+let heapProfilingTimer: NodeJS.Timeout | undefined = undefined
```

Comment:
@bryanhuhta I had to fix this as well. Sorry if this collides with your change – I'll rebase the branch, if you merge yours PR first

Reply:
Let's go with this change since it fixes a bug report!